python/cudf/setup.py

```diff
@@ -3,8 +3,9 @@
 from setuptools import find_packages
 from skbuild import setup
 
+packages = find_packages(include=["cudf*", "udf_cpp*"])
 setup(
-    include_package_data=True,
-    packages=find_packages(include=["cudf", "cudf.*"]),
+    packages=packages,
+    package_data={key: ["*.pxd", "*.hpp", "*.cuh"] for key in packages},
```

Comment:
It looks like you can specify `""` as a catch-all package (though the setup.cfg and pyproject.toml forms use `"*"` as the package name... wonder if that's correct or a typo.)
```suggestion
    package_data={"": ["*.pxd", "*.hpp", "*.cuh"]},
```

Reply:
You can do this, but it doesn't work correctly due to bugs in scikit-build.